src/ocm/components/clusterDetail/ClusterPollingErrorModal.tsx

```diff
@@ -1,21 +1,30 @@
 import React from 'react';
-import { Modal, ModalVariant } from '@patternfly/react-core';
+import { Button, ButtonVariant, Modal, ModalVariant } from '@patternfly/react-core';
 
 const ClusterPollingErrorModal = () => {
-  // TODO camador needs correct copy and to implement the retry action if there will be one
   const onClose = () => {
-    console.log('%c should close', 'font-size: 16px; color: red');
+    window.location.reload();
```

Comment:
We only want the modal to close when:
- a valid polling is received (The modal will close itself)
- the user clicks on reload the page

So it looks like we need to not show the close button, and we don't need an  `onClose` prop

Reply:
Ok, I've removed the close button